platforms/org.eclipse.mita.platform.arduinouno/src/org/eclipse/mita/platform/arduinouno/platform/ArduinoPlatformGeneratorModule.xtend

```diff
@@ -0,0 +1,23 @@
+package org.eclipse.mita.platform.arduinouno.platform
```

Comment:
It's not clear to me yet how users would use the code generated by this platform, particularly with the "SDK'/runtime part that they need to run the code.
There could either be documentation that describes the process or better yet, a Makefile generator which implements it.